config/locales/en.yml

```diff
@@ -30,7 +30,7 @@
 # available at https://guides.rubyonrails.org/i18n.html.
 
 en:
-  hello: "Hello world"
```

Comment:
We can revert the change en.yml, which is not relevant to this PR. not to pollute the history